BeforeBuild.sh

```diff
@@ -13,7 +13,16 @@ dotnet restore src/Sarif.Sdk.sln
 
 if [ $? != 0 ]
 then
-    echo nuget restore failed
+    echo nuget restore failed.
+    Exit
+    return $?
+fi
+
+dotnet msbuild /verbosity:minimal /target:BuildAndInjectObjectModel src/Sarif/Sarif.csproj /fileloggerparameters:Verbosity=detailed
```

Comment:
@michaelcfanning @jeff-bacon This will only work if the JSON Schema to .NET Code Generation Tool has been ported to .NET Core. Has it been?

Reply:
Linux is still not supported for SARIF NET Core.  jschema has yet to be ported to net core because of it's dependency on Microsoft.CodeAnalysis.  

---
In reply to: [147596572](https://github.com/Microsoft/sarif-sdk/pull/660#discussion_r147596572) [](ancestors = 147596572)